templates/docred/templates.yaml

```diff
@@ -0,0 +1,91 @@
+dataset: docred
+templates:
+  02af700c-e9e9-4a84-b75c-5fb29a5b7993: !Template
+    id: 02af700c-e9e9-4a84-b75c-5fb29a5b7993
+    jinja: "Read the following text and answer the questions.\n\nText:\n{% for sent\
+      \ in sents -%}\n{{ sent | join(\" \") }}\n{%- endfor %} \n\nQuestion:\nFrom\
+      \ the above text, find people, locations, organizations, time, numbers, and\
+      \ miscellaneous.\n|||\n{{\"People: \"}}\n{%- set people = [] -%} \n{%- for ners\
+      \ in vertexSet -%}\n{%- for ner in ners if ner['type'] == 'PER' -%}\n{{people.append(ner['name'])\
+      \ | default(\"\", True)}}\n{%- endfor -%}\n{%- endfor -%}\n{{ people | unique\
+      \ | join(\",\")}}{{\".\"}}\n\n{{\"Locations: \"}}\n{%- set locations = [] -%}\
```

Comment:
I think we want join(", ") not join(",") for tokenization reasons.

Reply:
hey sorry I made a new PR:

https://github.com/bigscience-workshop/promptsource/pull/268